release-calendar/src/client/components/Calendar.tsx

```diff
@@ -68,12 +70,19 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
           this.props.singleRelease,
         );
         this.setState({singleEvents: getSingleReleaseEvents(release)});
+        this.gotoDate(release[0].start);
       } else {
         this.setState({singleEvents: []});
+        this.gotoDate(new Date());
```

Comment:
Curious, does Full Calendar have a "go to Today" button option? Going back to the current date when you remove filters might be unexpected, so I'm not sure we should do this. But having a "go to Today" button would help!